lib/auth/Vault.js

```diff
@@ -1,4 +1,4 @@
-const errors = require('../errors');
+const errors = require('../errors').default;
```

Comment:
can't we stick with just importing errors simply:

const errors = require('../errors')

Reply:
I missed this was for JS only, and thus "temporary", while we have a cleaner way in typescript